### DISCUSSION TITLE
Support for giving different title name

### DISCUSSION THREAD

**Discussion by NitroSniper** [excalith/excalith-start-page#64] [Ideas]
Noticed that you have a small icon of your fetch in the tab which is really cool!

Made me wonder if you would add support for having a custom name for the title of the tab. This can be done by either having a "title" attribute in the data json. You can retain the current action if it is empty string or do something different.

**Comment by NitroSniper** [comment]
I'm glad that you also agree with that suggestion! About where to relocate the username attribute, I do believe that since we only see it in the prompt that it should be moved there since the prompt is part of the terminal. 

Thank you for the improvement of this!

**Reply by excalith** [reply]
Allrighty, prompt it is! Will convert this into enhancement request and will get back to it soon.

Cheers!

**Reply by excalith** [reply]
Hey again @NitroSniper, glad you liked that tiny little bit improvement! It really completes the customization part where you do not need to rely on the source assets. While implementing docker volumes, I also changed how we handle assets and saw a little improvement towards that favicon. I loved using a gif from Futurama and it immediately replaces favicon with that gif, looks great!

About your suggestion, it makes perfect sense to me. I think I will seperate the username variable and add another one for title. Username can be relocated into either prompt or terminal, and title can be seperated from the username completely. What do you think about where to put that username? Terminal sounds like more generic, like a session and makes sense to place, but prompt is the place wher users see their name most of the time. Which one would you prefer?

After this discussion, I will create a feature request from this discussion and make the change as soon as I can.

Cheers!

**Reply by excalith** [reply]
Hey @NitroSniper just released a version with a custom title option.
Cheers!
